OpenEdXMobile/res/layout/fragment_course_dashboard.xml

```diff
@@ -66,31 +67,30 @@
 
             <ImageView
                 android:id="@+id/header_play_icon"
-                android:layout_width="60dp"
-                android:layout_height="60dp"
+                android:layout_width="@dimen/course_card_play_icon_size"
+                android:layout_height="@dimen/course_card_play_icon_size"
                 android:layout_gravity="center"
                 android:clickable="false"
+                android:contentDescription="@string/go_to_video"
                 android:focusable="false"
                 android:focusableInTouchMode="false"
-                android:contentDescription="@string/go_to_video"
                 android:scaleType="fitXY"
                 android:src="@drawable/ic_media_play_inactive"
                 android:tint="@color/black"
-                android:visibility="invisible" />
+                android:visibility="invisible"
+                tools:visibility="visible" />
```

Comment:
Why are both android:visibility and tools:visibility set? Also, one is set to visisble and the other is set to invisisble.